_pytest/fixtures.py

```diff
@@ -1068,6 +1068,9 @@ def parsefactories(self, node_or_obj, nodeid=NOTSET, unittest=False):
         self._holderobjseen.add(holderobj)
         autousenames = []
         for name in dir(holderobj):
+            # Skip properties before the following getattr executes them.
+            if isinstance(getattr(type(holderobj), name, None), property):
```

Comment:
Perhaps a better way to look at the object is to iterate on `holderobj.__dict__` (getattr will involve the descriptor protocol, even if it's a class).